java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -3164,20 +3164,43 @@ public final ColumnVector clamp(Scalar lo, Scalar loReplace, Scalar hi, Scalar h
    * @param pattern Regex pattern to match to each string.
    * @return New ColumnVector of boolean results for each string.
    */
+  @Deprecated
   public final ColumnVector matchesRe(String pattern) {
+    return matchesRe(new RegexProgram(pattern, CaptureGroups.NON_CAPTURE));
+  }
+
+  /**
+   * Returns a boolean ColumnVector identifying rows which
+   * match the given regex program pattern but only at the beginning of the string.
+   *
+   * ```
+   * cv = ["abc", "123", "def456"]
+   * p = regex_program::create("\\d+")
```

Comment:
These are Javadocs but this is showing examples of C++ code.

Reply:
Updated. Thank you very much!